libs/ai-endpoints/langchain_nvidia_ai_endpoints/chat_models.py

```diff
@@ -381,18 +388,38 @@ def _generate(
         response = self._client.get_req(payload=payload, extra_headers=extra_headers)
         responses, _ = self._client.postprocess(response)
         self._set_callback_out(responses, run_manager)
-        parsed_response = self._custom_postprocess(responses, streaming=False)
+        parsed_response = self._custom_postprocess(
+            responses, streaming=False, stream_usage=False
```

Comment:
how about relying on streaming=False to imply stream_usage=False?

Reply:
what if it is streaming but do not want stream usage details?